Jenkinsfile

```diff
@@ -34,12 +57,9 @@ pipeline {
                         ])
                             
                         catchError(buildResult: 'FAILURE', stageResult: 'FAILURE') {
-                            bat """ 
-
-                            xcopy "..\\..\\ZELOOTD.z64" "OTRExporter\\"
-                            
-                            "${env.CMAKE}" -S . -B "build\\${env.PLATFORM}" -G "Visual Studio 17 2022" -T ${env.TOOLSET} -A ${env.PLATFORM} -D Python3_EXECUTABLE=${env.PYTHON} -D CMAKE_BUILD_TYPE:STRING=Release
-                            "${env.CMAKE}" --build ".\\build\\${env.PLATFORM}" --target ExtractAssets --config Release
+                            unstash 'assets'
```

Comment:
i know it shouldn't be an issue, but just want to make sure that there's nothing funky with using assets generated on linux for windows